include/logging/log_msg.h

```diff
@@ -22,6 +22,10 @@ extern "C" {
  * @{
  */
 
+#ifndef CONFIG_LOG_BLOCK_IN_THREAD_TIMEOUT_MS
+#define CONFIG_LOG_BLOCK_IN_THREAD_TIMEOUT_MS 0
```

Comment:
#define CONFIG_LOG_BLOCK_IN_THREAD_TIMEOUT_MS K_NO_WAIT

Reply:
i wouldn't do that. It is only to satisfy compiler when feature is disabled. Changing it to `K_NO_WAIT` could suggest that it is used. I'll add comment.